src/app/pages/landing.page/landing.page.component.ts

```diff
@@ -1,17 +1,44 @@
 import { Component, signal } from '@angular/core';
 import { HeroComponent } from '@components/hero/hero.component';
 import { LandingComponentVM } from './landing.component.vm';
+import { ProductComponent } from '@components/product/product.component';
+import { ProductItemComponent } from '@components/product/components/product-item/product-item.component';
+import { ProductListComponent } from '@components/product/components/product-list/product-list.component';
+import { AppRouteSegment } from 'src/app/app-route-segment';
 
 @Component({
   selector: 'app-landing.page',
   standalone: true,
-  imports: [HeroComponent],
+  imports: [HeroComponent, ProductComponent, ProductItemComponent, ProductListComponent],
   templateUrl: './landing.page.component.html',
 })
 export class LandingPageComponent {
   vm = signal<LandingComponentVM>({
     heroVM: {
       backgroundImageSrc: 'assets/farming.jpg',
     },
+    productListVM: {
+      productItems: [
```

Comment:
Please validate why having the items within the vm is necessary over content projection?